src/controllers/update/update.sign.out.controller.ts

```diff
@@ -2,16 +2,26 @@ import { NextFunction, Request, Response } from "express";
 import { createAndLogErrorRequest, logger } from "../../utils/logger";
 import * as config from "../../config";
 import { isActiveFeature } from "../../utils/feature.flag";
-import { JourneyType } from "../../model/data.types.model";
+import { JOURNEY_QUERY_PARAM, JourneyType } from "../../model/data.types.model";
+import { isRemoveJourney } from "../../utils/url";
 
 export const get = (req: Request, res: Response, next: NextFunction) => {
   try {
     logger.debugRequest(req, `${req.method} ${req.route.path}`);
+
+    let journey = JourneyType.update;
+    let previousPage = `${config.UPDATE_AN_OVERSEAS_ENTITY_URL}${req.query["page"]}`;
+
+    if (isRemoveJourney(req)) {
+      journey = JourneyType.remove;
+      previousPage += `?${JOURNEY_QUERY_PARAM}=${JourneyType.remove}`;
```

Comment:
You can use [this](https://github.com/companieshouse/overseas-entities-web/blob/0fb858dbcf6d86622cc8db34ad1fb539c4e1a403/src/config/index.ts#L182C14-L182C40) constant here.